test/integration/gardenlet/seed/seed/seed_test.go

```diff
@@ -115,6 +118,11 @@ var _ = Describe("Seed controller tests", func() {
 			seedNamespace = &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: gutil.ComputeGardenNamespace(seed.Name)}}
 			Expect(testClient.Create(ctx, seedNamespace)).To(Succeed())
 
+			By("wait till the manager obeserves seed namespace")
```

Comment:
Can you keep the `By` message consistent with the others?
```suggestion
			By("Wait until the manager obeserves seed namespace")
```